contrib/dtMediaWiki/dtMediaWiki.lua

```diff
@@ -0,0 +1,400 @@
+--[[dtMediaWiki is a darktable plugin which exports images to Wikimedia Commons
+    Author: Trougnouf (Benoit Brummer) <trougnouf@gmail.com>
+    Contributor: Simon Legner (simon04)
+
+Dependencies:
+* lua-sec: Lua bindings for OpenSSL library to provide TLS/SSL communication
+* lua-luajson: JSON parser/encoder for Lua
+* lua-multipart-post: HTTP Multipart Post helper
+]]
+local dt = require "darktable"
+local MediaWikiApi = require "contrib/dtMediaWiki/mediawikiapi"
```

Comment:
Please move mediawikiapi.lua into a lib subdirectory under the dtmediawiki directory.  script_manager ignores files in .../lib/... directories, figuring they are libraries and doesn't create a start button for them.

Reply:
Done, thank you !